src/dms/frontend/EdFi.DataManagementService.Frontend.AspNetCore/appsettings.json

```diff
@@ -23,7 +23,7 @@
         "OpenSearchUrl": "http://localhost:9200"
     },
     "DatabaseOptions": {
-        "IsolationLevel": "RepeatableRead"
+        "IsolationLevel": "ReadCommitted"
```

Comment:
Recent changes to the transaction handling made it safe to change to `ReadCommitted` as the default isolation level, but no one remembered to change that here.